### PR TITLE
Multi speaker

### PR DESCRIPTION
**Pull request by erogol**

### CODE REVIEW

**Comment by erogol** [comment]
@Edresson I'll wait your signal to merge this.

File: mozilla_voice_tts/tts/utils/generic_utils.py [hunk 1]
```diff
@@ -239,15 +248,33 @@ def check_config(c):
...
     check_argument('output_path', c, restricted=True, val_type=str)
 
-    # multi-speaker gst
+    # multi-speaker and gst
     check_argument('use_speaker_embedding', c, restricted=True, val_type=bool)
-    check_argument('style_wav_for_test', c, restricted=True, val_type=str)
+    check_argument('use_external_speaker_embedding_file', c, restricted=True, val_type=bool)
+    check_argument('external_speaker_embedding_file', c, restricted=True, val_type=str)
     check_argument('use_gst', c, restricted=True, val_type=bool)
+    check_argument('gst_style_input', c, restricted=True, val_type=str)
```
Comment (Edresson) [inline mozilla_voice_tts/tts/utils/generic_utils.py:256]: change this to:  
    check_argument('gst_style_input', c['gst'], restricted=True, val_type=str)

**Comment by Edresson** [comment]
@erogol  I reviewed the code and ran synthesis and training to test I found only 1 problem. I believe after solving it can merge :)

File: mozilla_voice_tts/tts/utils/generic_utils.py (hunk 1 above)
Comment (erogol) [reply mozilla_voice_tts/tts/utils/generic_utils.py]: I updated that part. There were multiple checks for the same arguments. Can you check again?

Comment (Edresson) [reply mozilla_voice_tts/tts/utils/generic_utils.py]: Now everything looks fine :)

**Pull request merged**